CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [Test] Add IAE test for deprecated edgeNGram analyzer name ([#5040](https://github.com/opensearch-project/OpenSearch/pull/5040))
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add feature flag for extensions ([#5211](https://github.com/opensearch-project/OpenSearch/pull/5211))
+- Added release notes for 1.3.7 ([#5418](https://github.com/opensearch-project/OpenSearch/pull/5418))
```

Comment:
You can omit a changelog entry for this, just apply the `skip-changelog` label to the PR to make the workflow pass. See details about when to add changelog entries in CONTRIBUTING.md#Changelog